verifier/src/error.rs

```diff
@@ -7,7 +7,7 @@ use mc_sgx_dcap_types::TcbError;
 use serde::{Deserialize, Serialize};
 
 /// Error working with quote evidence
-#[derive(displaydoc::Display, Debug, Clone, Serialize, Deserialize)]
+#[derive(displaydoc::Display, Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
needed to be able to add this error to https://github.com/mobilecoinfoundation/mobilecoin/blob/90560096c7cb306758b2d22bd4c5eb9891d20b66/attest/enclave-api/src/error.rs#L20 for the coming Dcap attestation logic